src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -290,9 +292,9 @@ public void execute() {
     /**
      * Helper implementation to perform execution.
      *
-     * @param checkstyleVersion Checkstyle compile version.
+     * @param checkstyleVersion Checkstyle compile version, may be null.
```

Comment:
What about this idea? 

Maybe the link is a little too much.

<img width="624" alt="image" src="https://github.com/user-attachments/assets/b2f46894-9026-4c01-92de-abfa07c474e2" />

<img width="835" alt="image" src="https://github.com/user-attachments/assets/dcd5862c-a36b-469a-b308-313abc986e76" />

```suggestion
     * @param checkstyleVersion {@link javax.annotation.Nullable} Checkstyle compile version.
```

Reply:
Hi @Pankraz76 , Thanks for the idea, we can discuss on that, Once this approach is finalized. Because these fix can be controversial. So at first we need to discuss whether annotation worth using or not.

Reply:
please revert comment update, better to not pollute comment wit htechnical details that might change, and comment wil be forgotten to be updated.